### DISCUSSION TITLE
Low quality approach circles in freeze frame

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21044] [Q&A] [answered]
the approach circles are really low quality when upscaled that much
as a skinning rule couldn't we make them vectors like some other scalable file type, i believe this is already a thing for some skin objects
![image](https://user-images.githubusercontent.com/78441708/199161664-a74373c9-cd32-417e-a422-969712bce791.png)
above image is with argon

this is also an issue with approach different
![image](https://user-images.githubusercontent.com/78441708/199161794-aa8127f7-9d3c-47e5-9af9-c93137f80111.png)
above image is also with argon

**Comment by peppy** [comment, marked as answer]
Eventually argon will be moved to non-sprite versions.

For legacy skins this will just be how they will look into the future. Newer skins can borrow from vector display methods.
